### ISSUE TITLE
Are the released pre-trained models the best models?

### ISSUE THREAD

**Issue by intellisense-team** [JUGGHM/PENet_ICRA2021#23]
Hi!
I just want to know whether the released pre-trained models the best models which can reproduce the results presented on the paper?

**Comment by JUGGHM** [comment]
Thanks for your interest! Yes they are the best models corresponding to the results in the paper. But I guess there could be still some space for improvement.

**Comment by KirkZhengYW** [comment]
Hi, @JUGGHM，I have tested the trained-models PENet in val_selection_cropped set, but the best RMSE is 757.197. Is there any extra configuration has to be made to reach the results on the KITTI benchmark?

**Comment by JUGGHM** [comment]
> Hi, @JUGGHM，I have tested the trained-models PENet in val_selection_cropped set, but the best RMSE is 757.197. Is there any extra configuration has to be made to reach the results on the KITTI benchmark?

Hi! 757.193 is the RMSE on the validation set. Empirically RMSE on the anonymous test set for bench-marking is around 20~30 mm lower.

**Comment by KirkZhengYW** [comment]
Thank you for your attention @JUGGHM .

**Comment by intellisense-team** [comment]
Thanks!!! @JUGGHM
